apps/www/src/config/customizer-items.ts

```diff
@@ -957,6 +957,23 @@ export const customizerItems: Record<string, SettingPlugin> = {
     badges: [customizerBadges.handler],
     route: customizerPlugins.media.route,
   },
+  [KEY_DELETE]: {
+    id: KEY_DELETE,
+    npmPackage: '@udecode/plate-select',
+    pluginFactory: 'createDeletePlugin',
+    pluginOptions: [
+      `options: {`,
+      `  query: {`,
+      `    allow: [`,
+      `      // ELEMENT_IMAGE, ELEMENT_HR`,
+      `    ],`,
+      `  },`,
+      `},`,
+    ],
+    label: 'Forward Delete Plugin',
+    badges: [customizerBadges.handler],
+    route: customizerPlugins.media.route,
+  },
```

Comment:
```suggestion
  [KEY_DELETE]: {
    id: KEY_DELETE,
    npmPackage: '@udecode/plate-delete',
    pluginFactory: 'createDeletePlugin',
    label: 'Delete',
    badges: [customizerBadges.handler],
  },
```

Reply:
Can you set the npm package to `-select` rather than `-delete`, since that's what we'll be releasing it under for now?

Reply:
I have made changes to above suggestion already.




> Can you set the npm package to `-select` rather than `-delete`, since that's what we'll be releasing it under for now?

package is npm-select only. am I missing something?

Reply:
It's showing as `-delete` in the code for me.
https://github.com/udecode/plate/pull/2687/files#diff-6760bf68e6f889f31c11b3678022f9f9f8355c05601800c959b5d588bcf53a59R962

Reply:
ah! gotcha, fixed, I directly took content from suggestions.